app/src/main/java/com/android/sample/ui/workout/RunningSelectionScreen.kt

```diff
@@ -0,0 +1,110 @@
+package com.android.sample.ui.workout
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.android.sample.R
+import com.android.sample.ui.composables.TopBar
+import com.android.sample.ui.navigation.NavigationActions
+import com.android.sample.ui.theme.Black
+import com.android.sample.ui.theme.Blue
+import com.android.sample.ui.theme.NeutralGrey
+
+@Composable
+fun RunningSelectionScreen(navigationActions: NavigationActions) {
+  val buttonColor = ButtonDefaults.buttonColors(Blue)
```

Comment:
Just use the color defined in our Color.kt

Reply:
That's what I did